CONTRIBUTING.md

```diff
@@ -151,40 +155,62 @@ Note: These instructions assume the user is using `mamba` instead of `conda` sin
 
   - **Note:** `mamba` should only be installed once in the base environment
 
+1. Setup env variables and clone the repo:
+   ```bash
+   export PYTHON_VER=3.8
+   export RAPIDS_VER=21.10
+   export CUDA_VER=11.4
+   export MORPHEUS_ROOT=$(pwd)/morpheus
+   git clone https://github.com/NVIDIA/Morpheus.git $MORPHEUS_ROOT
+   cd $MORPHEUS_ROOT
+   ```
 1. Create a new Conda environment
    ```bash
-   export CUDAToolkit_ROOT=/usr/local/cuda-{CUDA_VER}
-   mamba env create -n morpheus -f ./docker/conda/environments/cuda${CUDA_VER}_dev.yml
+   mamba create -n morpheus python=${PYTHON_VER}
    conda activate morpheus
    ```
-   This creates an environment named `morpheus` with all necessary dependencies, and activates that environment.
-2. Build cuDF
+
+   This creates a new environment named `morpheus`, and activates that environment.
+1. Set up `ssh-agent` to allow container to pull from private repos
+   ```bash
+   eval `ssh-agent -s`
+   ssh-add
+   ```
+1. Build cuDF
    ```bash
-   # Clone cuDF
-   git clone -b branch-${RAPIDS_VER} --depth 1 https://github.com/rapidsai/cudf ${MORPHEUS_ROOT}/.cache/cudf
-   cd ${MORPHEUS_ROOT}/.cache/cudf
-   # Apply the Morpheus cuDF patch
-   git apply --whitespace=fix ${MORPHEUS_ROOT}/cmake/deps/patches/cudf.patch
-   # Build cuDF libraries
-   ./build.sh --ptds libcudf cudf libcudf_kafka cudf_kafka
-   cd ${MORPHEUS_ROOT}
+   ./docker/build_conda_packages.sh libcudf cudf
+   mamba install -c file:///${MORPHEUS_ROOT}/.conda-bld -c nvidia -c rapidsai -c conda-forge libcudf cudf
    ```
    This will checkout, patch, build and install cuDF with the necessary fixes to allow Morpheus to work smoothly with cuDF DataFrames in C++.
-3. Build Morpheus
+1. Install remaining Morpheus dependencies
+   ```bash
+   mamba env update -n morpheus -f ./docker/conda/environments/cuda${CUDA_VER}_dev.yml
+   ```
+1. Build Morpheus
    ```bash
    ./scripts/compile.sh
    ```
    This script will run both CMake Configure with default options and CMake build.
-4. Install Morpheus
+1. Install Morpheus
    ```bash
    pip install -e ${MORPHEUS_ROOT}
    ```
    Once Morpheus has been built, it can be installed into the current virtual environment.
-5. Install camouflage, needed for the unittests
+1. Test the build (Note: some tests will be skipped)
+   ```bash
+   pytest
+   ```
+1. Optional: Run full end-to-end tests
+   - Our end-to-end tests require the [camouflage](https://testinggospels.github.io/camouflage/) testing framework. Install camouflage with:
+      ```bash
+      npm install -g camouflage-server
+      ```
+
```

Comment:
Note that this also requires nodejs to be installed

Reply:
I put it back in `docker/conda/environments/cuda11.4_dev.yml` in this PR